Polyhedron/demo/Polyhedron/MainWindow.cpp

```diff
@@ -1934,11 +1927,22 @@ void MainWindow::resetHeader()
   sceneView->header()->setSectionResizeMode(Scene::VisibleColumn, QHeaderView::Fixed);
   sceneView->header()->resizeSection(Scene::ColorColumn, sceneView->header()->fontMetrics().width("_#_"));
   sceneView->resizeColumnToContents(Scene::RenderingModeColumn);
-  if(sceneView->columnWidth(Scene::NameColumn) > fontMetrics().width(QString("This is a very long name")))
-    sceneView->header()->resizeSection(Scene::NameColumn, sceneView->header()->fontMetrics().width(QString("_This is a very long name_")));
-  else
-    sceneView->resizeColumnToContents(Scene::NameColumn);
   sceneView->header()->resizeSection(Scene::ABColumn, sceneView->header()->fontMetrics().width(QString("_AB_")));
   sceneView->header()->resizeSection(Scene::VisibleColumn, sceneView->header()->fontMetrics().width(QString("_View_")));
 
+  int name_width = sceneView->width() -
+      (sceneView->columnWidth(Scene::ColorColumn)
+      + sceneView->columnWidth(Scene::RenderingModeColumn)
+      + sceneView->header()->fontMetrics().width(QString("__AB__"))
+      + sceneView->columnWidth(Scene::VisibleColumn));
+    sceneView->header()->resizeSection(Scene::NameColumn, name_width);
+}
+
+bool MainWindow::eventFilter(QObject *, QEvent *event)
+{
+  if(event->type() == QEvent::Resize)
+  {
+    resetHeader();
+  }
+  return false;
```

Comment:
Apparently not. Even with ABColumn set to Fixed, it getd stretched and there is not enough space for the NameColumn. Plus, I don't remember exactly what Andreas specifically wanted when I started to change things in the sceneView, but I personally find it better to be able to resize the NameColumn as wanted by the user.